html/js/librenms.js

```diff
@@ -441,10 +417,10 @@ function destroy_map(id) {
 
 function populate_map_markers(map_id, group_radius = 10, status = [0,1], device_group = 0) {
     $.ajax({
-        type: "GET",
-        url: ajax_url + '/dash/worldmap',
+        type: "POST",
+        url: '/maps/getdevices',
```

Comment:
Urls like this won't work when people use subdirectories.

Reply:
Fixed